cdc/owner.go

```diff
@@ -333,13 +333,13 @@ func (o *Owner) newChangeFeed(
 	}
 	errCh := make(chan error, 1)
 
-	sink, err := sink.NewSink(ctx, id, info.SinkURI, filter, info.Config, info.Opts, errCh)
+	primarySink, err := sink.NewSink(ctx, id, info.SinkURI, filter, info.Config, info.Opts, errCh)
```

Comment:
```suggestion
	sink, err := sink.NewSink(ctx, id, info.SinkURI, filter, info.Config, info.Opts, errCh)
```

Reply:
I should use package `sink` in L360 (`var syncpointStore sink.SyncpointStore`)  to create a `syncpointStore`,if the var named sink,the package `sink` will not be used in the follow code,there is a conflict.

Reply:
Maybe move the `sink.SyncpointStore` to another package is better, but it reuses some code in the sink package.
We can refine this in another PR.